app/Http/Requests/BulkDownloadRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Not needed, authorization and validation can be performed directly in the controller